WinImageBuilder.psm1

```diff
@@ -644,6 +671,17 @@ function Compress-Image {
 }
 
 function Resize-VHDImage {
+    <#
+    .SYNOPSIS
+     This function resizes the VHD image to keep the storage requirements to the minimum size supported.

```

Comment:
This function resizes the VHD image to a minimum VHD size plus a FreeSpace parameter value buffer.